README.md

```diff
@@ -7,7 +7,7 @@
 [![](http://pkg.julialang.org/badges/YAML_0.7.svg)](http://pkg.julialang.org/?pkg=YAML)
 [![](http://pkg.julialang.org/badges/YAML_1.0.svg)](http://pkg.julialang.org/?pkg=YAML)
 
-[![Build Status](https://travis-ci.org/BioJulia/YAML.jl.svg?branch=master)](https://travis-ci.org/BioJulia/YAML.jl)
+[![Build Status](https://travis-ci.org/JuliaData/YAML.jl.svg?branch=master)](https://travis-ci.org/JuliaData/YAML.jl)
 [![Coverage Status](https://coveralls.io/repos/dcjones/YAML.jl/badge.svg?branch=master&service=github)](https://coveralls.io/github/dcjones/YAML.jl?branch=master)
```

Comment:
Looks like this coverage status is probably out of date? I prefer codecov myself.

Reply:
I'm sure that's true - should I just copy from CSV or something?

Reply:
I'll open an issue

Reply:
[![codecov](https://codecov.io/gh/JuliaData/YAML.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/JuliaData/YAML.jl)

Reply:
here's the codecov badge

Reply:
can be found here: https://codecov.io/gh/JuliaData/YAML.jl/settings/badge

Reply:
oh neat, I figured I'd have to do some additional configuration, but looks like it's already in the .travis.yaml 👍